programmers/난이도별/level01.시저_암호/HyeonJeong.py

```diff
@@ -0,0 +1,17 @@
+def solution(s, n):
+    answer = ""
+    for i in s:
+        if i == " ":
+            answer += " "
+        elif ('A' <= i <= 'Z' and ord(i) + n > 90) or ('a' <= i <= 'z' and ord(i) + n > 122):
+            answer += chr(ord(i) - 26 + n)
+        else:
+            answer += chr(ord(i) + n)
```

Comment:
```suggestion
        else:
            start = ord('a') if i.islower() else ord('A')
            answer += chr(((ord(i) - start + n) % 26) + start)
```

조금 더 직관적으로 풀이할 수 있을 것 같아요!

Reply:
저도 대문자, 소문자로 구분되어 돌아가게 만들고 싶었는데, 이걸 이렇게 해결해 주시네요! 고맙습니다

Reply:
@H-yeonJeong 일단 문제점을 발견하고 고민을 시작하는 것이 중요합니다! 그 이후는 경험이나 개인 역량의 차이니까요 ㅎㅎ 시작이 중요하답니다!! 앞으로 꾸준히 풀이하다보면 자연스럽게 고민에 대한 해답을 찾아가리라 생각해요!
화이팅!